Aztec/Classes/Formatters/Implementations/MarkFormatter.swift

```diff
@@ -24,8 +24,8 @@ class MarkFormatter: AttributeFormatter {
     func remove(from attributes: [NSAttributedString.Key: Any]) -> [NSAttributedString.Key: Any] {
         var resultingAttributes = attributes
 
+        resultingAttributes[.foregroundColor] = placeholderAttributes![.foregroundColor]
```

Comment:
Are we sure `placeholderAttributes` is safe to force unwrap every time or should we do the `guard let` pattern?